.github/actions/environment/action.yml

```diff
@@ -51,14 +42,19 @@ runs:
       run: >
         dotnet workload install \
           maui-android \
-          ${{ runner.os == 'macOS' && 'maui-ios maui-maccatalyst maui-windows maui-tizen' || '' }} \
+          ${{ runner.os == 'macOS' && 'maui-ios maui-maccatalyst maui-windows' || '' }} \
           ${{ runner.os == 'Windows' && 'maui-windows' || '' }} \
           --temp-dir "${{ runner.temp }}" --from-rollback-file rollback.json
 
-    # We build Sentry.Maui for every supported MAUI target so we can access platform-specific features.
-    # That includes Tizen. We don't need the entire Tizen SDK, but we do need the base Tizen workload.
-    # Though we could install it on all runners, we really only need it when building and packing for NuGet, which we do on macOS only.
-    - name: Install Tizen Workload
-      if: runner.os == 'macOS'
+    - name: Restore workloads
+      shell: bash
+      run: dotnet workload restore
+
+    - name: Update workloads
+      shell: bash
+      run: dotnet workload update
```

Comment:
Can we include an explanation for this? We just ran `dotnet workload restore` right? Won't doing `dotnet workload update` give us non-deterministic builds (we don't know what version of the workloads is going to be used for the build)?